cli/progress/multiprogress_test.go

```diff
@@ -225,12 +228,42 @@ func TestMultiProgressBarOrdering(t *testing.T) {
 	}()
 	err = multiprogress.Open()
 	require.NoError(t, err)
-	assert.Contains(t, buf.String(), "test1                   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% score: F")
-	assert.Contains(t, buf.String(), "test2                   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% score: F")
-	assert.Contains(t, buf.String(), "test3                   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% score: F")
-	assert.Contains(t, buf.String(), "3/3 scanned             ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100%")
+	assert.Contains(t, buf.String(), "test1                           ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% score: F")
+	assert.Contains(t, buf.String(), "test2                           ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% score: F")
+	assert.Contains(t, buf.String(), "test3                           ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% score: F")
+	assert.Contains(t, buf.String(), "3/3 scanned                     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100%")
 	// regexp is not working, perhaps because of ansi escape characters???
 	// ordering := regexp.MustCompile(`^.*test1.*test3.*test2.*$`)
 	// m := ordering.FindString(buf.String())
-	assert.Contains(t, buf.String(), "test1                   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% score: F\r\n test3                   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% score: F\r\n test2")
+	assert.Contains(t, buf.String(), "test1                           ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% score: F\r\n test3                           ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% score: F\r\n test2")
+}
+
+func TestMultiProgressBarNotApplicable(t *testing.T) {
+	var in bytes.Buffer
+	var buf bytes.Buffer
+
+	progressBarElements := map[string]string{"1": "test1", "2": "test2", "3": "test3"}
+	multiprogress, err := newMultiProgressBarsMock(progressBarElements, []string{"1", "2", "3"}, &in, &buf)
+	require.NoError(t, err)
+
+	go func() {
+		// we need to wait for tea to start the Program, otherwise these would be no-ops
+		time.Sleep(1 * time.Millisecond)
+		multiprogress.OnProgress("1", 1.0)
+		multiprogress.Score("1", "F")
+		multiprogress.Completed("1")
+
+		multiprogress.Score("2", "X")
+		multiprogress.Errored("2")
+
+		multiprogress.Score("3", "U")
+		multiprogress.NotApplicable("3")
+	}()
+	err = multiprogress.Open()
+	require.NoError(t, err)
+
+	assert.Contains(t, buf.String(), "test1                           ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% score: F")
+	assert.Contains(t, buf.String(), "test2                           ───────────────────────────────────    X score: X")
+	assert.Contains(t, buf.String(), "test3                           ───────────────────────────────────  N/A score: U")
+	assert.Contains(t, buf.String(), "1/3 scanned 1/3 errored 1/3 n/a ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100%")
```

Comment:
Now, that I see this output and independent of this PR:
Should we perhaps change "scanned" to "scored"? Also not 100% correct, as "X" is also a score, but perhaps more intuitive?

Reply:
I think scored is not the correct term for cnquery. Lets change this as soon as we find a better term